CRM/Member/Form.php

```diff
@@ -77,6 +77,23 @@ class CRM_Member_Form extends CRM_Contribute_Form_AbstractEditPayment {
   public $_priceSetId;
 
   /**
+   * Used when price set specified.  Used to indicate that form is only to retrieve
+   * price set.  How we use price set is unconventional.  We re-use the same
+   * Membership/MembershipRenewal Form used to enter memberships.  When user selects
+   * a PriceSet, we use JavaScript to inject HTML returned by a recursive call to this form.
+   *
+   * This flag controls this recursive call, short-circuiting it to only retrieve the price
```

Comment:
@alifrumin, @eileenmcnaughton has talked me into holding off the PR until at the very least unit tests can be done.  I think, ideally testing would cover a more finished set of changes.  So sounds like you'll be holding off a little while still.

Until then, I will manually pull the PR on every customer upgrade (yuk).